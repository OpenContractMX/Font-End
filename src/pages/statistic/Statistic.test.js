import React from "react";
import "@testing-library/jest-dom/extend-expect";
import { render } from "@testing-library/react";
import { Statistic } from "./";

it("[render content `statistic / Statistic`]", () => {
  const filter = { category: "Categoria", year: "AÃ±o" };
  const contractsChars = {
    contracts_number: "0",
    inversion: 0,
    top_ten: [
      {
        id: 0,
        month: 0,
        trimester: 0,
        title: "vacio",
        buyer_name: "vacio",
        date: "0",
        amount: 0,
        currency: "",
      },
    ],
    execution_mean: 0,
    months: {
      1: 0,
      2: 0,
      3: 0,
      4: 0,
      5: 0,
      6: 0,
      7: 0,
      8: 0,
      9: 0,
      10: 0,
      11: 0,
      12: 0,
    },
  };
  const { asFragment } = render(
    <Statistic contractsChars={contractsChars} filter={filter} />
  );
  expect(asFragment()).toMatchSnapshot();
});
