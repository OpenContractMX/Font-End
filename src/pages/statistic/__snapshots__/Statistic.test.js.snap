// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[render content \`statistic / Statistic\`] 1`] = `
<DocumentFragment>
  <main
    class="container-statistic"
  >
    <section
      class="filter-category"
    >
      <h2
        class="filter-category--title"
      >
        Filtrar:
      </h2>
      <div
        class="filter-category__forms"
      >
        <form
          class="filter-category__forms--filter-category"
          for="filter-category"
        >
          <label>
            Categoria:
          </label>
          <select
            class="filter-category__forms--filter-category--select-category"
          >
            <option
              value="Categoría"
            >
              Categoría
            </option>
            <option
              value="seguridad"
            >
              Seguridad
            </option>
            <option
              value="salud"
            >
              Salud
            </option>
            <option
              value="energia"
            >
              Energía
            </option>
            <option
              value="economia"
            >
              Economía
            </option>
            <option
              value="gobernacion"
            >
              Gobernación
            </option>
            <option
              value="medio_ambiente"
            >
              Medio Ambiente
            </option>
            <option
              value="educacion"
            >
              Educación
            </option>
            <option
              value="comunicacion_transporte"
            >
              Comunicación y Transporte
            </option>
            <option
              value="social"
            >
              Social
            </option>
            <option
              value="investigacion"
            >
              Investigación
            </option>
            <option
              value="trabajo"
            >
              Trabajo
            </option>
          </select>
        </form>
        <p>
          Y
        </p>
        <form
          class="filter-category__forms--filter-year"
          for="filter-year"
        >
          <label>
            Año:
          </label>
          <select
            class="filter-category__forms--filter-year--select-year"
          >
            <option
              value="Año"
            >
              Año
            </option>
            <option
              value="2015"
            >
              2015
            </option>
            <option
              value="2016"
            >
              2016
            </option>
            <option
              value="2017"
            >
              2017
            </option>
            <option
              value="2018"
            >
              2018
            </option>
          </select>
        </form>
      </div>
    </section>
    <p
      class="container-statistic--no-filter"
    >
      Filtra por categoria y año para obtener datos 
    </p>
  </main>
</DocumentFragment>
`;
